packages/cli/src/config/settingsSchema.ts

```diff
@@ -101,7 +101,7 @@ export const SETTINGS_SCHEMA = {
     label: 'Enable Usage Statistics',
     category: 'General',
     requiresRestart: true,
-    default: false,
+    default: true,
```

Comment:
Note: this was already the case. We also have a `/privacy` command that allows interaction with this as well